light/provider/http/http_test.go

```diff
@@ -91,3 +95,41 @@ func TestProvider(t *testing.T) {
 	require.Contains(t, err.Error(), "connection refused")
 	require.Nil(t, lb)
 }
+
+// TestLightClient_NilCommit ensures correct handling of a case where commit returned by http client is nil
+func TestLightClient_NilCommit(t *testing.T) {
+	chainID := "none"
+	c := &rpcmock.RemoteClient{}
+	p := lighthttp.NewWithClient(chainID, c)
+	require.NotNil(t, p)
+
+	c.On("Commit", testify.Anything, testify.Anything).Return(
```

Comment:
suggestion: what do you think about following the rule where the chain call has to place on a new line if the entire expression takes more than one line?